monitoring/oplog-populator/dashboard.py

```diff
@@ -0,0 +1,335 @@
+        legendFormat='{{connector}}',
+    )])
+
+max_pipeline_size = Stat(
+    title='Max Pipeline Size Per Connector',
+    description='Maximum size of the mongo pipeline per connector',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='0',
+    reduceCalc='last',
+    format=UNITS.BYTES,
+    orientation='horizontal',
+    graphMode='none',
+    targets=[Target(
+        expr='sum by(connector) (' + Metrics.PIPELINE_SIZE.raw() + ')',
+        legendFormat='{{connector}}',
+    )],
+    thresholds=[
+        Threshold('green', 0, 0.),
+    ])
+
+bucket_additions = TimeSeries(
+    title='Bucket Addition Rate',
+    description='Rate at which buckets are getting added to the connectors',
+    dataSource='${DS_PROMETHEUS}',
+    gradientMode='opacity',
+    lineInterpolation='smooth',
+    fillOpacity=10,
+    spanNulls=True,
+    unit=UNITS.OPS_PER_SEC,
+    targets=[Target(
+        expr='sum by(connector) (rate(' + Metrics.CONNECTOR_CONFIG(opType="add") + '))',
+        legendFormat='{{connector}}',
+    )])
+
+bucket_removals = TimeSeries(
+    title='Bucket Removal Rate',
+    description='Rate at which buckets are getting removed from the connectors',
+    dataSource='${DS_PROMETHEUS}',
+    gradientMode='opacity',
+    lineInterpolation='smooth',
+    fillOpacity=10,
+    spanNulls=True,
+    unit=UNITS.OPS_PER_SEC,
+    targets=[Target(
+        expr='sum by(connector) (rate(' + Metrics.CONNECTOR_CONFIG(opType="delete") + '))',
+        legendFormat='{{connector}}',
+    )])
+
+ack_lag = TimeSeries(
+    title='Acknowledgement Lag By Operation Type',
+    description='Delta between a config change in mongo and it getting processed by the oplogPopulator in seconds',
+    dataSource='${DS_PROMETHEUS}',
+    gradientMode='opacity',
+    lineInterpolation='smooth',
+    fillOpacity=10,
+    spanNulls=True,
+    unit=UNITS.SECONDS,
+    targets=[Target(
+        expr='sum by(opType) (rate(' + Metrics.ACK_LAG.sum() + '))',
+        legendFormat='{{opType}}',
+    )])
+
+reconfig_lag = TimeSeries(
+    title='Reconfig Lag By Connector',
+    description='Time it takes to reconfigure a connector in seconds',
+    dataSource='${DS_PROMETHEUS}',
+    gradientMode='opacity',
+    lineInterpolation='smooth',
+    fillOpacity=10,
+    spanNulls=True,
+    unit=UNITS.SECONDS,
+    targets=[Target(
+        expr='sum by(connector) (rate(' + Metrics.RECONFIG_LAG.sum() + '))',
+        legendFormat='{{connector}}',
+    )])
+
+bucket_per_connector = TimeSeries(
+    title='Buckets Per Connector',
+    description='Evolution of buckets per connector',
+    dataSource='${DS_PROMETHEUS}',
+    gradientMode='opacity',
+    lineInterpolation='smooth',
+    fillOpacity=10,
+    spanNulls=True,
+    targets=[Target(
+        expr='sum by(connector) (' + Metrics.BUCKETS.raw() + ')',
+        legendFormat='{{connector}}',
+    )])
+
+request_body_size_per_connector = TimeSeries(
+    title='Request Body Size Per Connector',
```

Comment:
this does not match the snapshot in the PR message ;)
Please update the snapshot, and remember it will actually be useful when you need to provide screenshots for documentation!